src/pages/faq.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import NavigationBar from "../components/Layout/NavigationBar";
+import FaqContent from "../components/FAQ/FaqContent";
+import FooterBar from "../components/Layout/FooterBar";
+
+export const FaqPage = () => (
+  <>
+    <NavigationBar active="faq" />
+    <FaqContent />
+    <FooterBar />
+  </>
+);
```

Comment:
I would prefer to make a common component with fixed component ie. footer and header and pass the changing component as a child.

Reply:
created an issue => #14, out of scope